pkg/tools/k8sutils/k8sutils.go

```diff
@@ -285,14 +301,14 @@ func GetWorkflowByName(name string) (*serverlessv1alpha1.Workflow, bool, error)
 		return nil, false, nil
 	}
 	var wf *serverlessv1alpha1.Workflow
-	switch obj.(type) {
+	switch obj := obj.(type) {
```

Comment:
assigning the result of this type assertion to a variable could eliminate type assertions in switch cases (S1034)